src/units/sensor.cpp

```diff
@@ -100,6 +100,7 @@ int TimeSecretType[2][MAX_TIME_SECRET] = {{UVS_ITEM_TYPE::PEELOT,UVS_ITEM_TYPE::
 
 int ThreallMessageProcess;
 extern uvsVanger *Gamer;
+int sensorStaticFrame = 0; // animation speed control frame, required to skip animation frames on high fps

```

Comment:
We also should skip frames if it too slow because it can produce strange bugs like show maps upside-down.

Reply:
consequences like this https://github.com/KranX/Vangers/commit/212836ed04a7648ee9e7596e001f5e76565fb817#diff-924ce6850d040aaafbf5972340732df5R3334